src/redux/connectApi/connect.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+
+const connectURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const bookId = 'Zj9BJJLgZixblvJ42n1G';
+
+const ConnectBook = {
```

Comment:
- [ ] I think this is not the best naming convention in React. For utility functions, camelCase should be use. So instead of `ConnectBook` :x: , we should use `connectBook` :heavy_check_mark: